components/members/detail/ActivityBadge.tsx

```diff
@@ -0,0 +1,57 @@
+import styled from '@emotion/styled';
+import { FC, useEffect, useState } from 'react';
+
+import { Activity } from '@/api/members/type';
+import { getProjectById } from '@/api/projects';
+import { categoryLabel } from '@/components/projects/upload/constants';
+import { MOBILE_MEDIA_QUERY } from '@/styles/mediaQuery';
+import { textStyles } from '@/styles/typography';
+
+const ActivityBadge: FC<Activity> = (activity) => {
+  const [category, setCategory] = useState('');
+  const getProjectCategory = async () => {
+    if (activity.isProject) {
+      const project = await getProjectById(activity.id.toString());
+      setCategory(categoryLabel[project.category]);
+    }
+  };
+
+  useEffect(() => {
+    getProjectCategory();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
```suggestion
  useEffect(() => {
    const getProjectCategory = async () => {
    if (activity.isProject) {
      const project = await getProjectById(activity.id.toString());
      setCategory(categoryLabel[project.category]);
    }
  };
    getProjectCategory();
  }, []);
```

`useEffect` 안에 외부의 함수, 변수를 쓴다면 의존성으로 써주는게 더 좋은 것 같아! 
글고 이 경우는 어차피 `useEffect` 안쪽에만 쓰이니까 이렇게 옮겨도 좋겠당~